isis/src/qisis/objs/JigsawRunWidget/JigsawRunWidget.cpp

```diff
@@ -267,14 +259,135 @@ namespace Isis {
     }
     else {
       // Make sure to abort the bundle if it is currently running.
-      m_bundleAdjust->abortBundle();
-      m_bRunning = false;
       m_ui->JigsawRunButton->setText("&Aborting...");
+      m_ui->statusOutputLabel->setText("Aborting...");
```

Comment:
You may have already asked or talked to Ken about this, but do we want to discard the output on the left side if we abort a bundle.

For example, if a user is running a bundle and they see something in the left hand output where they want to abort the bundle, it will change to "aborting..." and discard the iteration outputs. Do we want that or do we want to keep the iteration outputs?